### PR TITLE
Delete wrong use RuntimeHelpers.OffsetToStringData

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: MonoGame.Framework/Graphics/OpenGL.cs [hunk 1]
```diff
@@ -1522,7 +1522,7 @@ protected unsafe static IntPtr MarshalStringToPtr (string str)
             if (intPtr == IntPtr.Zero) {
                 throw new OutOfMemoryException ();
             }
-            fixed (char* chars = str + RuntimeHelpers.OffsetToStringData / 2) {
+            fixed (char* chars = str) {
```
Comment (jkotas) [inline MonoGame.Framework/Graphics/OpenGL.cs]: The original code was actually doing something like this:

`fixed (char* chars = String.Concat(str, (RuntimeHelpers.OffsetToStringData / 2).ToString()))`

**Comment by tomspilman** [comment]
NOTE: This is related to the discussion in https://github.com/MonoGame/MonoGame/pull/6132.

**Comment by tomspilman** [comment]
@dellis1972 - So this change builds correctly and seems like the correct code to me.  Why should we not merge it?

**Comment by tomspilman** [comment]
This same fix has already been applied:

 https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Platform/Graphics/OpenGL.cs#L1574

Closing this.

**Comment by jkotas** [comment]
> This same fix has already been applied:

@tomspilman I do not see the fix in the current tip. The bug is still there.

**Pull request closed without merging**